manila/share/manager.py

```diff
@@ -4042,7 +4042,8 @@ def do_deferred_snapshot_deletion(self, ctxt):
                   "process their deletion.")
```

Comment:
remove this periodic task

Reply:
as disussed in call, periodic task are for error_deferred_deleting.